bower.json

```diff
@@ -0,0 +1,22 @@
+
+{
+  "name": "jquery-oembed-all",
+  "version": "0.9RC1",
+  "homepage": "https://github.com/nfl/jquery-oembed-all",
+  "authors": [
+    "National Football League <engineers@nfl.com>"
+  ],
+  "description": "Use native oEmbed services when possible.",
+  "main": "jquery.oembed.js",
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
```

Comment:
I think this needs a couple Updates 

- `jquery` should be included as a `dependency`
- `node_modules`, `bower_components` and  `test(s)` don't need to be ignored cause they don't exist in this module. You should instead ignore `examples/`, `*.md`, and `CHANGELOG`.